app/views/pages/ect_additional_information.html.erb

```diff
@@ -0,0 +1,161 @@
+<% content_for :title, "ECF induction and training: additional information for early career teachers" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-xl">ECF induction and training: additional information for early career teachers</h1>
+    <p>
+      This guidance is for early career teachers (ECTs). It answers some common questions about your 2-year induction programme,
+      which is based on the early career framework (ECF).
+    </p>
+    <p>
+      Find out how to register for your school’s ECF-based training programme, what happens during your induction, and how to get help.
+    </p>
+    <p>You can also read:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        <%= govuk_link_to "guidance for school leaders, including the different roles involved in your induction",
+            "school-leader-additional-information"
+            %>
+      </li>
+      <li>
+        <%= govuk_link_to "guidance for mentors",
+            "mentor-additional-information"
+            %>
+      </li>
+      <li>
+        <%= govuk_link_to "ECF reforms: overview",
+            "https://www.gov.uk/government/publications/early-career-framework-reforms-overview"
+            %>
+      </li>
+    </ul>
+
+    <%= render ContentListComponent.new(heading: "Contents",
+                                        list: [
+                                              "Is the ECF an assessment tool, and can I fail my training?",
+                                              "Do I have to complete ECF-based training on top of another induction programme?",
+                                              "Do I need to collect evidence during my ECF-based training?",
+                                              "What happens if I start mid-term, work part-time, or I’m serving a reduced induction?",
+                                              "What should I do if I have difficulties with my induction programme?",
+                                              "How do I register with the DfE’s service?"
+                                              ]) %>
+
+    <section id="Is-the-ECF-an-assessment-tool,-and-can-I-fail-my-training?">
```

Comment:
```suggestion
    <section id="is-the-ecf-an-assessment-tool-and-can-i-fail-my-training">
```

This would be much cleaner with `parameterize`!

Reply:
Definitely!